ast/ast.go

```diff
@@ -70,6 +70,7 @@ type Node interface {
 	Type() string // Type property is a string that contains the variant type of the node
 	Location() SourceLocation
 	Errs() []Error
+	CommentsList() []Comment
```

Comment:
Nitpick: `CommentList`.

Reply:
Thanks for pointing that out, I have fixed it.